app/assets/javascripts/discourse/app/helpers/topic-status-icons.js

```diff
@@ -9,7 +9,7 @@ export default ArrayProxy.extend({
       renderIcon(name, key);
     };
 
-    if (topic.get("closed") && topic.get("archived")) {
+    if (topic.closed && topic.archived) {
```

Comment:
The topic model is still an EmberObject, so I think we'll need to keep these `.get()` calls if we want autotracking to work. (which I think we do, since `glimmer-header/topic/status` now uses this function in a native getter)